tox_core/src/state_format/old.rs

```diff
@@ -749,86 +735,4 @@ mod tests {
             },
         ])
     );
-
-    encode_decode_test!(
```

Comment:
Why not keep it?

Reply:
encode_decode_test compares result with `Eq`, but it's not implemented for secret and precomputed keys.